### DISCUSSION TITLE
Is there a Linux equivalent for CrashDumps?

### DISCUSSION THREAD

**Discussion by danipen** [dotnet/runtime#65715] [Q&A] [answered]
Hi, I'm trying to debug a crash in native code in my .net core 6 app.

When running under windows platform, when application crashes it creates a minidump (.dmp) file in:
`C:\Users\xxx\AppData\Local\CrashDumps`

Is there something similar in Linux platform? I tried to find the minidump files in linux, but I was unable to find them.

**Comment by jkotas** [comment, marked as answer]
The equivalent on Linux is called core dump. You can either let the operating system create the core dump of the crashed process by running `ulimit -c unlimited`, or you can let the .NET runtime itself collect the core dump by setting environment variables described in https://docs.microsoft.com/en-us/dotnet/core/diagnostics/dumps . The later option has configuration similar to Windows minidumps, but it works only when the .NET runtime is able to intercept the process crash.
